scripts/docker_cmake

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+rm -f  CMakeCache.txt
+rm -rf CMakeFiles/
```

Comment:
don't some apps have a `make distclean` or `make realclean` option that does this?